### PR TITLE
docs(bench): add scalar benchmarks for integer

### PR DESCRIPTION
**Pull request by IceTDrinker**

### CODE REVIEW

File: tfhe/docs/getting_started/benchmarks.md [hunk 1]
```diff
@@ -61,7 +77,7 @@ Using the same m6i.metal machine as the one for tfhe-rs, the timings are:
```
Comment (soonum) [inline tfhe/docs/getting_started/benchmarks.md:77]: The machine is not m6i.metal anymore

File: tfhe/docs/getting_started/benchmarks.md [hunk 2]
```diff
@@ -84,8 +100,8 @@ Using the same m6i.metal machine as the one for tfhe-rs, the timings are:
```
Comment (soonum) [inline tfhe/docs/getting_started/benchmarks.md:100]: Same here.

**Review by soonum — CHANGES_REQUESTED** [review]
Still checking the numbers.

File: tfhe/docs/getting_started/benchmarks.md (hunk 1 above)
Comment (IceTDrinker) [reply tfhe/docs/getting_started/benchmarks.md:77]: done

File: tfhe/docs/getting_started/benchmarks.md (hunk 2 above)
Comment (IceTDrinker) [reply tfhe/docs/getting_started/benchmarks.md:100]: done

**Review by soonum — APPROVED** [review]
All checked and good to go !

**Pull request merged**